### ISSUE TITLE
Alt+UpArrow/DownArrow conflicts with new shortcuts in Python console

### ISSUE THREAD

**Issue by CyrilleB79** [mltony/nvda-sentence-nav#18]
### Issue
From NVDA 2021.1 onwards, Alt+UpArrow and Alt+DownArrow are the default shortcut to jump to previous/next result in NVDA's console output. This conflicts with next/previous sentence shortcut of this add-on.

### Solution
I guess sentence navigation is not useful in the console output area.
Thus I would recommend to deactivate sentence navigation in the console's output panel.

### Alternative solution
Leave thins as is and ask the user to modify themselves the conflicting shortcuts, either in Sentence Nav or in console output. IMO it is a degraded UX and is less desirable.

### Note
I have also tried to put "nvda" in the applications where Sentence Nav should be deactivated (in the add-on's options). But this does not seem to work. Did I do something wrong? Or is NVDA a special case?

**Comment by mltony** [comment]
Where can I read more about this? I just tried the latest v2021.1 RC2 
and in Python console Alt+Up and Alt+Down don't seem to do anything.

Adding NVDA to the blacklist of apps should work. I can try to debug why 
it's not working, but I need to be able to reproduce this on my computer 
before I can do so.

**Comment by CyrilleB79** [comment]
These new capabilities are documented in the Change log of 2021.1:
> * Easier navigation of output in NVDA Python Console. (#9784)
> * alt+up/down jumps to the previous/next output result (add shift for selecting).
> * control+l clears the output pane.

According to the PR that has introduced these shortcuts, it should also be documented in the dev doc according to the PR. But the dev doc for my 2021.1rc2 strangely seems to be the one of 2020.4.

Moreover, I have tried again to put NVDA in the black-list and I understand why it is not useful in this case. Sentence Nav is actually disabled in the console if you black-list NVDA.
But the new console shortcuts are not defined by the application (i.e. wx). Instead, they are NVDA scripts with the same assignation keys as Sentence Nav ones. Thus gesture.send() cannot work.

**Comment by CyrilleB79** [comment]
To test these new scripts:
* disable the whole Sentence Nav add-on
* open the console
* Execute various commands, e.g. `help(api)` or `2+3` etc.
* Press F6 to go to the output pane
* press alt+upArrow to go to previous results

**Comment by mltony** [comment]
Makes sense. I just fixed this in the latest commit. Thanks for flagging!

**Comment by CyrilleB79** [comment]
Tested last version of master.
I confirm it is working. Thanks!

**Comment by cary-rowen** [comment]
The new version has been released, the issue can be closed
